manifests/init.pp

```diff
@@ -116,6 +119,46 @@
     $rancid_path_env_real = $rancid_path_env
   }
 
+  if $vcs == 'USE_DEFAULTS' {
+    $vcs_real = $default_vcs
+  } else {
+    validate_re($vcs, '^(cvs|svn|git)$', "rancid::vcs is ${vcs} and must be one of [cvs, svn, git]")
+    $vcs_real = $vcs
+  }
+
+  if $vcsroot == 'USE_DEFAULTS' {
+    case $vcs_real {
+      default: {
+        notify { "Rancid does not supported vcs ${vcs_real}.": }
+      }
+      'cvs': {
+        $vcsroot_real = '$BASEDIR/CVS'
+      }
+      'svn': {
+        $vcsroot_real = '$BASEDIR/svn'
+      }
+      'git': {
+        $vcsroot_real = '$BASEDIR/.git'
+      }
+    }
+  } else {
+    $vcsroot_real = $vcsroot
+  }
+
+  # Debian and RedHat (currently) use the same names for these packages, so no
+  # need to switch on osfamily
+  case $vcs_real {
```

Comment:
These packages are very likely managed by other modules. Would suggest adding a parameter `manage_packages` set to false and wrap that conditional around the package resource below.